Spring-server/src/main/java/ikpi63holding/postclient/LoginViewModel.java

```diff
@@ -0,0 +1,22 @@
+package ikpi63holding.postclient;
+
+public class LoginViewModel {
+    private String username;
+    private String password;
+
+    public String getUsername() {
+        return username;
+    }
+
+    public String getPassword() {
+        return password;
+    }
+
+    public void setUsername(String username) {
+        this.username = username;
+    }
+
+    public void setPassword(String password) {
+        this.password = password;
+    }
+}
```

Comment:
Is this file necessary? We have already **User.java**, you could use some fields of them by auth process

Reply:
Oh, i forgot about User class.